README.md

```diff
@@ -70,6 +70,19 @@ Set the maximum number of images to be displayed `m` or `--max` flag. 0 means in
 | z OR Left Click  | Toggle actual size vs scaled image                     |
 | . (period)       | Repeat last action                                     |
 
+### Command Mode Controls
+
+
+| Command                     | Action                                                   |
+|-----------------------------|----------------------------------------------------------|
+| ng OR newglob [glob]        | **Required argument** the new glob/directory/file        |
+| ? OR help                   | Toggle fullscreen mode                                   |
```

Comment:
Toggle help menu?

Reply:
fuck that, I can fix it directly. Let's just get this merged.